.github/workflows/build-sw-container.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and Push Docker image
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - /^v([0-9.]+)$/
```

Comment:
It looks like regular expressions aren't supported so you'll need to modify this using https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet to make sure that `v1.2.3`-style tags trigger the run. Note that any part of the version number can have multiple digits

Reply:
Must the tag have two dots? For example, are v1.2 and v2 valid?

Reply:
> It looks like regular expressions aren't supported so you'll need to modify this using https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet to make sure that `v1.2.3`-style tags trigger the run. Note that any part of the version number can have multiple digits

Must the tag have two dots? For example, are v1.2 and v2 valid?

Reply:
We use something approaching [semantic versioning](https://semver.org/) so there will always be 3 numbers each separated by 1 dot so `v1.2` and `v2` are invalid in this case.

Reply:
Will `v[0-9]+.[0-9]+.[0-9]+` work?

Reply:
Looks right to me!

Reply:
No, we've occasionally used 4 numbers. You could try `v[0-9.]+` or `v[0-9]+.[0-9]+.[0-9]+*` -- both are a little too permissive but I don't see a way to get it exactly right.

Reply:
Since `tags` is a list we could do something like:

```
tags:
  - v[0-9]+.[0-9]+.[0-9]+
  - v[0-9]+.[0-9]+.[0-9]+.[0-9]+
```

Reply:
Sure, that works.